### PR TITLE
feat: Release process for TP1

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

File: releases/tp1_release.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Release Syndesis TP1
+
+This is the release plan for TP1, which consist of multiple distinct repos. This plan changes when going to a Monorepo, as many of the projects below then share the same version number (and releasy cycle)
```
Comment (jimmidyson) [inline releases/tp1_release.md]: typo: `releasy` -> `release`

File: releases/tp1_release.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## Tagging and Building
+
+### External Images
+
+* https://github.com/syndesisio/pemtokeystore
+* https://github.com/syndesisio/keycloak-openshift
+
+Those have fixed version numbers are not necessarily released afressh. They have
```
Comment (jimmidyson) [inline releases/tp1_release.md]: typo: `afressh` -> `afresh`

File: releases/tp1_release.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+# Release Syndesis TP1
+
+This is the release plan for TP1, which consist of multiple distinct repos. This plan changes when going to a Monorepo, as many of the projects below then share the same version number (and releasy cycle)
+
+## Tagging and Building
+
+### External Images
+
+* https://github.com/syndesisio/pemtokeystore
```
Comment (jimmidyson) [inline releases/tp1_release.md:9]: Not need for TP2 though

File: releases/tp1_release.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+* https://github.com/syndesisio/keycloak-openshift
```
Comment (jimmidyson) [inline releases/tp1_release.md:10]: Not need for TP2 though

File: releases/tp1_release.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
+# Release Syndesis TP1
+
+This is the release plan for TP1, which consist of multiple distinct repos. This plan changes when going to a Monorepo, as many of the projects below then share the same version number (and releasy cycle)
+
+## Tagging and Building
+
+### External Images
```
Comment (jimmidyson) [inline releases/tp1_release.md]: Missing oauth proxy?

**Review by jimmidyson — CHANGES_REQUESTED** [review]

**Comment by rhuss** [comment]
Added  a tp2_release.md document specific for tp2 (and keeping the tp1 document for historical reasons). 

There is still the issue how we switch over templates from TP1 to TP2 (so only after QE is done). I suggest to add the version to the template name (e.g. `syndesis-dev-restricted-1.1.0`) and when TP2 should be activated then we tell OSO support to update the provisioning script to use this templates name. That way we can easily switch back and force between TPs.

@jimmidyson wdyt ? (this would need some update to the generator though)

File: releases/tp1_release.md (hunk 4 above)
Comment (rhuss) [reply releases/tp1_release.md:10]: left it here for historical reason, removed from TP2 document.

File: releases/tp1_release.md (hunk 3 above)
Comment (rhuss) [reply releases/tp1_release.md:9]: left it here for historical reason, removed from TP2 document.

**Comment by rhuss** [comment]
@jimmidyson I would opt for a merge, please.

**Comment by rhuss** [comment]
@jimmidyson hope you dont mind the manual merge, I will update anything left afterwards.

**Pull request merged**